src/config/tabs.rs

```diff
@@ -148,20 +155,34 @@ impl From<PaneTypeFile> for PaneType {
             PaneTypeFile::TabContent => PaneType::TabContent,
             #[cfg(debug_assertions)]
             PaneTypeFile::FrameCount => PaneType::FrameCount,
-            PaneTypeFile::Property { content: properties, align, scroll_speed } => {
-                PaneType::Property {
-                    content: properties
-                        .into_iter()
-                        .map(|prop| prop.try_into().expect(""))
-                        .collect_vec(),
-                    align: align.into(),
-                    scroll_speed,
-                }
-            }
+            PaneTypeFile::Property { content, align, scroll_speed, format } => PaneType::Property {
+                content: {
+                    match (format, content) {
+                        (Some(format), Some(_)) | (Some(format), None) => parser::parser()
+                            .parse(&format)
+                            .into_result()
+                            .map_err(|e| {
+                                anyhow::anyhow!("Failed to parse property format: {:?}", e)
+                            })?
+                            .into_iter()
+                            .map(|prop| -> Result<_> { prop.try_into() })
+                            .try_collect()?,
```

Comment:
This should be documented, users should know that format will take precedence, or it might spark confusion